src/components/CommentList.jsx

```diff
@@ -1,19 +1,52 @@
 import {React, useEffect, useState} from 'react'
-import ArticleSummary from './ArticleSummary'
-import { getCommentsByArticleId } from '../../api'
-import { useParams } from 'react-router-dom'
+import {getCommentsByArticleId, createComment} from '../../api'
+import {useParams} from 'react-router-dom'
 import '../css/Comment.css'
 
 
-function CommentList() {
+function CommentList({comments, setComments} ) {
+    // State. 
+    const [inputValue, setInputValue] = useState('')
+    const [isSendingComment, setIsSendingComment] = useState(false)
+
+     // ToDo - ultimately allow to pick from DB or add actual sign-up/authentication functionality
+    const defaultUsername = 'grumpy19'
+
     const {article_id}= useParams()
-    const [comments, setComments] = useState([])
 
     // Functions.  
     async function getCommentsSetState(articleId) {
         const commentsForArticle = await getCommentsByArticleId(articleId)
         setComments(commentsForArticle)
       }
+    
+    async function handleSubmit(event) {
+        event.preventDefault()
+        setIsSendingComment(true)
+        const commentBody = event.target[0].value
+
+        // Empty the textarea so we get the placeholder back / can't resubmit the comment.
```

Comment:
All these comments are great to see your thought process, but just make sure you delete them before pushing code for review 👍

Reply:
Really? Isn't this sort of thing helpful for any developer who will look at this code in the future? 
Maybe this particular one is a bit OTT? It wasn't obvious to me that you would have to manually clear submitted data to redisplay the placeholder text.